nvflare/private/fed/server/server_engine.py

```diff
@@ -208,6 +209,9 @@ def start_app_on_server(self, run_number: str, job_id: str = None, job_clients=N
             self.engine_info.status = MachineStatus.STARTING
             app_custom_folder = workspace.get_app_custom_dir(run_number)
 
+            job_id = None
+            if job:
```

Comment:
Can the job ever be None? If it is, then job_id would be None, then will self._start_runner_process() work with a None job_id?

Reply:
There's an old "start_app server" command calling this, which has been deprecated. I will remove that code, then no need to this case.